src/cocoa/toga_cocoa/widgets/box.py

```diff
@@ -1,7 +1,7 @@
 from rubicon.objc import *
```

Comment:
At column 1: (F403) 'from rubicon.objc import *' used; unable to detect undefined names